openapi-contracts/src/test/java/com/sinch/sdk/domains/verification/models/dto/v1/VerificationReportDtoTest.java

```diff
@@ -0,0 +1,90 @@
+package com.sinch.sdk.domains.verification.models.dto.v1;
+
+import com.adelean.inject.resources.junit.jupiter.GivenJsonResource;
+import com.adelean.inject.resources.junit.jupiter.TestWithResources;
+import com.sinch.sdk.BaseTest;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@TestWithResources
+public class VerificationReportDtoTest extends BaseTest {
+
+  @GivenJsonResource("/domains/verification/v1/VerificationReportCalloutResponseDto.json")
+  VerificationResponseDto loadedVerificationCalloutDto;
+
+  public static VerificationResponseDto expectedVerificationCalloutDto =
+      new VerificationResponseDto()
+          .id("the id")
+          .method(VerificationMethodDto.CALLOUT)
+          .status("FAIL")
+          .reason("Fraud")
+          .callComplete(true)
+          .reference("my reference")
+          .price(
+              new VerificationResponsePriceDto(
+                  new VerificationPriceInformationDto()
+                      .verificationPrice(
+                          new VerificationPriceInformationVerificationPriceDto(
+                              new MoneyDto()
+                                  .currencyId("verificationPrice currency id")
+                                  .amount(3.141516F)))
+                      .terminationPrice(
+                          new VerificationPriceInformationTerminationPriceDto(
+                              new MoneyDto()
+                                  .currencyId("terminationPrice currency id")
+                                  .amount(6.626070F)))
```

Comment:
Another question about the price: as we have 2 different decimal prices, do you think it would be worth to provide a helper method that returns the total price (taking care about the issues related to summing floats in java)?

Reply:
We could imagine such helper; but should be common to all SDKs (having same level of service between them)
This kind of helper/feature should, may be, provided according to users future feedbacks

Reply:
Indeed, it was a wild guess, I don't know if it's a valid use case 🤷‍♂️